ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1206,18 +1206,18 @@ private System.Collections.IEnumerator
                 closest_physical_parent(part1).flightID);
           }
           foreach (var collider in part1.currentCollisions) {
-            if (collider.gameObject.layer == (int)UnityLayers.LocalScenery) {
-              Log.Info("Reporting collision with local scenery " +
-                       collider.name);
-              plugin_.ReportGroundCollision(
-                  closest_physical_parent(part1).flightID);
-            }
```

Comment:
This code was odd: let's use `collider` and later check if by any chance it was null...